playhouse/db_url.py

```diff
@@ -42,6 +51,11 @@ def parseresult_to_dict(parsed):
     elif 'sqlite' in parsed.scheme and not connect_kwargs['database']:
         connect_kwargs['database'] = ':memory:'
 
+    # Get additional connection args from the query string
+    connect_kwargs.update(
```

Comment:
Agreed. I suffered from tunnel vision here, caused by the late hour and having previously used a dict comprehension to parse the results of parse_qs . . . followed by the Python 2.6 problems.